server-phoenix/lib/helios_web/schema/resolvers/widget_collection.ex

```diff
@@ -17,6 +17,8 @@ defmodule HeliosWeb.Schema.Resolvers.WidgetCollection do
   end
 
   def by_id_or_first(_parent, %{id: id}, _info) do
-    {:ok, Enum.find(Jason.decode!(WidgetCollectionData.enabled), fn widget -> widget["id"] === id end)}
+    decodedData = Jason.decode!(WidgetCollectionData.enabled)
+    idExists = Enum.find(decodedData, fn widget -> widget["id"] === id end)
```

Comment:
Nit: I think convention in elixir is snake case for variable names.

A unit test for this bug fix would also be great to prevent regressions when we connect the database.